lazy-pages/src/utils.rs

```diff
@@ -66,35 +68,55 @@ pub fn with_inclusive_ranges<P: PageU32Size + Ord, E>(
     f(iter)
 }
 
-#[test]
-fn test_with_inclusive_range() {
-    use gear_core::memory::GearPage;
+pub(crate) fn handle_psg_case_one_page(
```

Comment:
Consider make it `LazyPagesExecutionContext` method

Reply:
done